cytoscape-qtip.js

```diff
@@ -52,6 +52,10 @@ SOFTWARE.
     }
   };
 
+  var fixDomContainer = function(ele) {
+    return ele && ( ele[0] === window ) ? undefined : ele;
```

Comment:
Would it be more natural to default on `document.body` instead of nothing for `window`?  Maybe it would make sense to print a `console.warn()` message in any case so the consumer knows that `window` isn't generally supported?

Reply:
> Would it be more natural to default on `document.body` instead of nothing for `window`?

The idea was to separate this method (its purpose is just to fix `$(window)` input for jQuery 3 compatibility)  from assigning `document.body` as a default in `generateOpts` method. It seems logical, but on the other hand I don't know if it's correct for such a tiny library where the method `fixDomContainer` is used only 2 times. I'd leave it as it is, but the final decision is on you.

> Maybe it would make sense to print a `console.warn()` message in any case so the consumer knows that `window` isn't generally supported?

It's a sound idea, I'd do this, the only possible (but unconvincing) counter argument is that this PR is just a workaround because `qTip2` maintainer does not accept similar PR there, and no `console.warn()` messages were in that PR.